include/grpc/grpc_security.h

```diff
@@ -564,6 +566,9 @@ GRPCAPI grpc_server_credentials*
 grpc_ssl_server_credentials_create_with_options(
     grpc_ssl_server_credentials_options* options);
 
+GRPCAPI void grpc_ssl_server_credentials_set_send_client_ca_list(
```

Comment:
I think this method should operate on `grpc_ssl_server_credentials_options`, not directly on `grpc_server_credentials`.

Reply:
Done.